actions/dump-charm-debug-artifacts/action.yml

```
@@ -39,10 +43,20 @@ runs:
 
     # Save logs
 
+    - name: Export artifact prefix
+      shell: bash
+      run: |
+        if [ -n "${{ inputs.artifact-prefix }}" ];
+        then
+          echo "prefix=${{ github.job }}-${{ inputs.artifact-prefix }}" >> "$GITHUB_ENV"
```

suggestion (non-blocker):
```suggestion
          echo "prefix=${{ inputs.artifact-prefix }}-${{ github.job }}" >> "$GITHUB_ENV"
```
This way it'd be easier for us to identify the jobs, sometimes the name of the job is long so the prefix would be drowned by it. WDYT?

---

makes sense, visualizing it helps, it looks ok to me :+1: